peps/pep-9999.rst

```diff
@@ -0,0 +1,616 @@
+PEP: 9999
+Title: Standardizing selection of Python package files across multiple sources
```

Comment:
I think we can come up with a snazzier title that both makes the PEP easier to find, and captures the audiences attention.  I'd like to at least add the word "priority" and am not sure we need "Standarizing" as that's what the PEP process is inherently all about.

Reply:
See what you think. I'm not sure we'll make it to "snazzy," but I hope this is closer